### PR TITLE
Trailing camera alvinleung

### PR DESCRIPTION
**Pull request by alvinleung**

### CODE REVIEW

File: Assets/Scripts/CameraController.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    private Vector3 offsetBetweenTarget;
+    private Vector3 velocity;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        Vector3 pointingDirection = ((this.transform.rotation * Vector3.forward));
+
+        offsetBetweenTarget = -pointingDirection * followDistance;
+        //ffsetBetweenTarget = this.transform.position - GetAvgSubjectsPosition();
```
Comment (thomasnakagawa) [inline Assets/Scripts/CameraController.cs]: Please remove this commented out code if it is not needed

File: Assets/Scripts/CameraController.cs [hunk 2]
```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CameraController : MonoBehaviour
+{
+    [SerializeField] List<GameObject> followingSubjectList;
```
Comment (thomasnakagawa) [inline Assets/Scripts/CameraController.cs]: Private uninitialized fields give a compiler warning in the editor, consider initializing this with `followingSubjectList = default`

**Review by thomasnakagawa — APPROVED** [review]
Nice job! The camera behavior looks smooth, I left a couple comments suggesting ways to make the code cleaner, otherwise the code looks good!

**Comment by alvinleung** [comment]
@thomasnakagawa How to resolve that mac .DS_Store conflict?

**Comment by jleung51** [comment]
@alvinleung Since `.DS_Store` is in the `.gitignore` file you can probably just remove it from the repo and it won't be added or tracked in the future!

**Pull request merged**